v2/manifests/core-ons/zebedee.yml

```diff
@@ -49,10 +49,13 @@ services:
       website_reindex_key:            "1hZiEDeZcVKZwO6WmTDTDhVSiRAKS0jM6Nzlvlszk0OW0vY5M2FCiGD7ncqcucxB"
       scheduled_publishing_enabled:   "false"
       OTEL_JAVAAGENT_ENABLED:         ${OTEL_JAVAAGENT_ENABLED:-"false"}
-      ELASTIC_SEARCH_CLUSTER:         "elasticsearch"
-      ELASTIC_SEARCH_SERVER:          "elasticsearch"
+      ELASTIC_SEARCH_CLUSTER:         "cmdelasticsearch"
+      ELASTIC_SEARCH_SERVER:          "cmdelasticsearch"
       brian_url:                      "http://project-brian:8083"
       BABBAGE_URL:                    "http://babbage:8080"
+      UPLOAD_SERVICE_API_URL:         "http://dp-upload-service:25100"
+      ENABLE_UPLOAD_NEW_ENDPOINT:     true
+      DEFAULT_WEBSITE_URL:            "http://babbage:8080"
```

Comment:
I'm not sure you need to set this, might be wrong.

Reply:
These have now been removed - a hangover from some previous work